Code/Tools/ProjectManager/Source/ProjectButtonWidget.cpp

```diff
@@ -331,6 +346,25 @@ namespace O3DE::ProjectManager
         QDesktopServices::openUrl(m_projectInfo.m_logUrl);
     }
 
+    void ProjectButton::SetEngine(const EngineInfo& engine)
+    {
+        m_engineInfo = engine;
+        m_engineNameLabel->SetText(m_engineInfo.m_name + " " + m_engineInfo.m_version);
+        m_engineNameLabel->update();
+        m_engineNameLabel->setObjectName(m_engineInfo.m_thisEngine ? "thisEngineLabel" : "otherEngineLabel");
+        m_engineNameLabel->setToolTip(m_engineInfo.m_name + " " + m_engineInfo.m_version + " " + m_engineInfo.m_path);
+        m_engineNameLabel->refreshStyle(); // important for styles to work correctly
```

Comment:
Doesn't Qt have a [QStandardItemModel](https://doc.qt.io/qt-6/model-view-programming.html) that can update the view when a particular entry within the model changes?

Though that is probably overkill for just a button.

Reply:
Yes it does! and we should total be using a model more as we scale up the number of objects and improve performance,  it's on the backlog.